homeassistant/util/unit_system.py

```diff
@@ -234,6 +254,12 @@ def _deprecated_unit_system(value: str) -> str:
             for unit in UnitOfPressure
             if unit != UnitOfPressure.HPA
         },
+        # Convert non-metric area
+        ("area", UnitOfArea.SQUARE_INCHES): UnitOfArea.SQUARE_MILLIMETERS,
```

Comment:
A mm² is two orders of magnitude smaller than an in2, would cm² be a better fit for the default conversion?

Reply:
Imperial units are weird but yeah, will update that.